packages/puppeteer-core/src/node/ChromeLauncher.ts

```diff
@@ -43,6 +44,27 @@ export class ChromeLauncher extends ProductLauncher {
     super(puppeteer, 'chrome');
   }
 
+  override launch(options: PuppeteerNodeLaunchOptions = {}): Promise<Browser> {
+    const headless = options.headless ?? true;
+    if (
+      headless === true &&
+      this.puppeteer.configuration.logLevel !== 'silent'
+    ) {
+      console.warn(
+        [
+          '\u001b[1m\u001B[43m\u001b[30m',
+          'Puppeteer Headless Deprecation Warning:\u001b[0m\u001b[33m',
```

Comment:
Let's say "Old headless" and let's not captitilize every letter in the title probably